tests/tracer/test_ci.py

```diff
@@ -126,17 +126,55 @@ def test_git_extract_workspace_path_error(tmpdir):
 
 def test_extract_git_metadata(git_repo):
     """Test that extract_git_metadata() sets all tags correctly."""
-    expected_tags = {
-        git.REPOSITORY_URL: "git@github.com:test-repo-url.git",
-        git.COMMIT_MESSAGE: "this is a commit msg",
-        git.COMMIT_AUTHOR_NAME: "John Doe",
-        git.COMMIT_AUTHOR_EMAIL: "john@doe.com",
-        git.COMMIT_AUTHOR_DATE: "2021-01-19T09:24:53-0400",
-        git.COMMIT_COMMITTER_NAME: "Jane Doe",
-        git.COMMIT_COMMITTER_EMAIL: "jane@doe.com",
-        git.COMMIT_COMMITTER_DATE: "2021-01-20T04:37:21-0400",
+    extracted_tags = git.extract_git_metadata(cwd=git_repo)
+
+    assert extracted_tags["git.repository_url"] == "git@github.com:test-repo-url.git"
+    assert extracted_tags["git.commit.message"] == "this is a commit msg"
+    assert extracted_tags["git.commit.author.name"] == "John Doe"
+    assert extracted_tags["git.commit.author.email"] == "john@doe.com"
+    assert extracted_tags["git.commit.author.date"] == "2021-01-19T09:24:53-0400"
+    assert extracted_tags["git.commit.committer.name"] == "Jane Doe"
+    assert extracted_tags["git.commit.committer.email"] == "jane@doe.com"
+    assert extracted_tags["git.commit.committer.date"] == "2021-01-20T04:37:21-0400"
+    assert extracted_tags["git.branch"] == "master"
```

Comment:
Can a check also be included for `git.commit.sha`?

Reply:
I don't think we can set the commit hash in the test git environment, if you're asking about setting it to some value and checking here for the same value. Otherwise we would just test by reproducing the same code we use to read the hash from git and I'm not sure if that's ideal... what do you think?

Reply:
A check that it exists and is not empty would be enough I think.